go-controller/pkg/ovn/namespace.go

```diff
@@ -47,9 +47,11 @@ func (oc *Controller) addPodToNamespace(ns string, portInfo *lpInfo) error {
 	}
 	defer nsInfo.Unlock()
 
-	// DUAL-STACK FIXME: handle multiple IPs
-	if err := nsInfo.addressSet.AddIP(portInfo.ips[0]); err != nil {
-		return err
+	//FIXME DUAL-STACK
+	if len(portInfo.ips) > 0 {
```

Comment:
is this a separate fix not related to implementing IPAM? If so, can you make it into another commit please?

Reply:
it's part of rebasing with master to make dual stack work.

Reply:
that being said, i will be rehashing the commits later once the PR looks ok from code perspective.